telemetry_core/src/main/java/com/newrelic/telemetry/spans/Span.java

```diff
@@ -12,7 +12,10 @@
 import com.newrelic.telemetry.util.Utils;
 import lombok.Value;
 
-/** TODO: Document me */
+/**
+ * Spans are New Relic's analog of OpenTracing spans. They can represent events such as external
+ * call,s individual operations, and datastore calls.
```

Comment:
a comma got misplaced on this line. 
Also, it might be good to mention that a collection of spans with the same `traceId` make up a distributed-tracing Trace.